Expensify/App
# Do not include CP merge commits in the list of PR merge commits

### Details


We are having several issues which are caused by the fact that when PR is CP'ed to staging, this creates a new commit in git history, which starts the same as every other merge commit ie: `Merge pull request #4444 `.

This duplicate then causes multiple problems, see linked issue for details.

### Fixed Issues

$ https://github.com/Expensify/App/issues/4977

### Tests


Updated the GitUtilsTest.js to have unit tests

### QA Steps


Once deployed to production, we will need to monitor for example staging deploy message to see, if the pull requests, which have been cherry-picked to staging will appear in two deploy checklists in a row or if the PRs will get deploy messaged for two consequent deploys. They should not, the PR should only be considered as part of the deploy into thich it was CP'ed.

### Tested On

Unit tests.
